classes/02class/exercises/c02-network01/Faria-Marcio/c02-network01.txt

```diff
@@ -0,0 +1,7 @@
+IP			IP in binary				subnet in binary			subnet in decimals	is it private 	how many IP addresses?	
+																or public?	
+10.0.0.1/24		00001010.00000000.00000000.00000001	11111111.11111111.11111111.00000000	255.255.255.0		private		254	

```

Comment:
if you considered AWS environment instead of a regular network, hosts = 251, because AWS reserves 5 IP address per subnet

```
10.0.0.0: Network address.

10.0.0.1: Reserved by AWS for the VPC router.

10.0.0.2: Reserved by AWS. The IP address of the DNS server is the base of the VPC network range plus two. For VPCs with multiple CIDR blocks, the IP address of the DNS server is located in the primary CIDR. We also reserve the base of each subnet range plus two for all CIDR blocks in the VPC. For more information, see Amazon DNS Server.

10.0.0.3: Reserved by AWS for future use.

10.0.0.255: Network broadcast address. We do not support broadcast in a VPC, therefore we reserve this address.
```